mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -11,7 +11,7 @@ use crate::{mm2::lp_stats::{add_node_to_version_stat, remove_node_from_version_s
             mm2::rpc::lp_commands::{get_public_key, get_public_key_hash, get_shared_db_id, trezor_connection_status}};
 use coins::eth::EthCoin;
 use coins::my_tx_history_v2::my_tx_history_v2_rpc;
-#[cfg(feature = "enable-nft-integration")] use coins::nft;
+use coins::nft;
```

Comment:
future note: we will need to create it's own crate for nft

Reply:
> will need to create it's own crate for nft

should think about it, in the future there are swaps with nfts. wouldn't it be strange that nfts are not a part of the coins in this case?

Reply:
It can inherit from coins, but it's not really a `coin`. Majority part of the coins modules needs to be seperated to different/their own crates. e.g `eth` module has amazingly much than what it should have. Which takes development/maintanince/compilation costs to much higher levels.